src/app/(auth)/(components)/LinkedInLoginButton/LinkedInLoginButton.tsx

```diff
@@ -0,0 +1,32 @@
+'use client'
+import GithubIcon from '@/assets/icons/GithubIcon'
+import { Button } from '@/components/Button/Button'
+
+import { signIn } from 'next-auth/react'
+import { useState } from 'react'
+import styles from '../GithubLoginButton/GithubLoginButton.module.scss'
+
+interface GithubLoginButtonProps {
+  label?: string
+}
+
+export const LinkedInLoginButton = (props: GithubLoginButtonProps) => {
+  const [isCalled, setIsCalled] = useState(false)
+  return (
+    <Button
+      disabled={isCalled}
+      onClick={() => {
+        setIsCalled(true)
+        signIn('linkedin', {
+          callbackUrl: '/api/auth/callback/linkedi',
```

Comment:
I noticed a small typo in the line 21: shouldent it be: `callbackUrl: '/api/auth/callback/linkedin'` ?

Reply:
Yes You are right. This is legacy PR. 
I managed to get LinkedIn auth working but I don't know if it's on GitHub yet.

Reply:
Good. Thanks for reply :)